bin/virtme-ssh-proxy

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import argparse
+import array
+import socket
+
+
+def get_hostname_cid(ssh_dst):
+    """Return the hostname and CID
+
+    >>> hostname, cid = get_hostname_cid("virtme-ng/24")
+    >>> hostname
+    'virtme-ng'
+    >>> cid
+    24
+    >>> hostname, cid = get_hostname_cid("virtme/ng/21")
+    >>> hostname
+    'virtme/ng'
+    >>> cid
+    21
+
+    """
+    splitted = ssh_dst.rsplit("/", 1)
+    hostname, cid = splitted[0], int(splitted[1], 10)
+    return (hostname, cid)
+
+
+# See https://docs.python.org/3/library/socket.html#socket.socket.sendmsg
+def send_fds(sock, fds):
+    ancdata = [(socket.SOL_SOCKET, socket.SCM_RIGHTS, array.array("i", fds))]
+    sock.sendmsg(
+        [b"\0"],
+        ancdata,
+        socket.MSG_CMSG_CLOEXEC,
+    )
+
+
+def passfds(cid, port):
+    sock = socket.socket(socket.AF_VSOCK, socket.SOCK_STREAM | socket.SOCK_CLOEXEC, 0)
+    sock.connect((cid, port))
+    return send_fds(socket.socket(fileno=1), [sock.fileno()])
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog="virtme-connect",
+    )
+    parser.add_argument("ssh_destination")
+    parser.add_argument("-p", "--port", type=int, default=22)
+    args = parser.parse_args()
```

Comment:
just to make this script lighter and simply, why not simply using `sys.argv[0]` and `sys.argv[1]` and not using `argparse`?

In the SSH config, you can simply pass the fellowing:

```
ProxyCommand "{ssh_proxy}" %h %p
```

Reply:
> just to make this script lighter and simply, why not simply using `sys.argv[0]` and `sys.argv[1]` and not using `argparse`?

About this, if we want to add more options in the future argparse gives more flexibility and the overhead compared to sys.argv[] should be negligible. If performance becomes an issue, we can always rewrite this in Rust later.

Reply:
From what I saw, there are not many other parameters you need to pass to the proxy :)
And that's controlled by us so only using the positions should be fine, no?

Reply:
Yeah, it can be considered a "backend" tool, not frontend, so positional is fine as well. I guess I can rephrase my comment as "both ways are fine". :)

Reply:
See comment https://github.com/arighi/virtme-ng/pull/261#discussion_r2030216141